db/migrations/20210318214325_create_clip_kick.sql

```diff
@@ -0,0 +1,33 @@
+-- +goose Up
+CREATE TABLE maker.clip_kick
+(
+    id         SERIAL PRIMARY KEY,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id BIGINT  NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    bid_id     NUMERIC NOT NULL,
+    top        NUMERIC,
+    tab        NUMERIC,
+    lot        NUMERIC,
+    usr        TEXT,
```

Comment:
`usr` and `kpr` should probably be `BIGINT` here in keeping with the convention of having addresses stored as IDs.